### ISSUE TITLE
Missing files when sampling with pre-trained model

### ISSUE THREAD

**Issue by Suimingzhe** [fdbtrs/IDiff-Face#3]
Thanks for your job!
Could you please provide the file './hydra' and 'ema_averaged_model.ckpt' in pre-trained CPD25 and CPD50 ?

**Comment by NeRF-Factory** [comment]
After so many hours of debugging...I finally run the code successfully. 
The  './hydra' and 'ema_averaged_model.ckpt' are unnecessary. Instead, I modify several lines of code in 'sample. py'. If you want the code, you can add my WeChat:130008841490. I will provide a detailed explanation for you.

**Comment by aksy2512** [comment]
@NeRF-Factory Can you please explain, how did u change the sampling code

**Comment by NeRF-Factory** [comment]
@aksy2512 
Exactly, the checkpoint provided by the official repository is the weight of the 'eps_model' which is defined in the class 'DenoisingDiffusionProbabilisticModel'. You should write a new code for loading the checkpoint in the "sample.py". Additionally, the sampling process is also defined in 'DenoisingDiffusionProbabilisticModel'. You can find the code in "./diffusion/ddpm.py". This requires some modifications, but it's not too difficult.

I hope this help to answer your concern.

**Comment by robbbgm** [comment]
@NeRF-Factory Could you please provide us with the code?

**Comment by fdbtrs** [comment]
The instructions for sampling have been updated. Check the requirements list and  make sure to download all required models and configuration
